test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs

```diff
@@ -41,11 +43,33 @@ void Initialize()
             text.WidthResizePolicy = ResizePolicyType.FillToParent;
             mainWin.Add(text);
 
+            List<Window.WindowOrientation> list = new List<Window.WindowOrientation>();
+
+            list.Add(Window.WindowOrientation.Landscape);
+            list.Add(Window.WindowOrientation.LandscapeInverse);
+            list.Add(Window.WindowOrientation.NoOrientationPreference);
+            list.Add(Window.WindowOrientation.Portrait);
+            list.Add(Window.WindowOrientation.PortraitInverse);
+
+            mainWin.SetAvailableOrientations(list);
+
             Animation animation = new Animation(2000);
             animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
             animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
             animation.Looping = true;
             animation.Play();
+
+            tm = new Timer(100);
+            tm.Tick += Tm_Tick;
+            tm.Start();
+        }
+
+        private bool Tm_Tick(object source, Timer.TickEventArgs e)
+        {
+            bool rotating = mainWin.IsWindowRotating();
+            log.Fatal(tag, $"window is Rotating: {rotating}");
+            log.Fatal(tag, "window is Rotating: {}"+rotating);
```

Comment:
I think we dont need to reply this informations twice.

Reply:
Ah..Thank you for detail review :)